packages/variable-resolver/src/browser/variable-quick-open-service.ts

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (C) 2018 Red Hat, Inc.
```

Comment:
I think the copyright should be under Apache 2 license:
```
/*
 * Copyright (C) 2018 Read Hat, Inc. and others.
 *
 * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
 */
```
cc @svenefftinge

Reply:
I've updated all files